pypeit/calibrations.py

```diff
@@ -413,6 +413,25 @@ def get_bias(self):
 
         # Save & return
         self._update_cache('bias', 'bias', self.msbias)
+
+        # If we need to make a bad pixel mask using the bias frames, do it now
+        if self.par['badpix']:
```

Comment:
Am trying to figure out why this comes here, given we should have
already have created a `bpm`.  Consider removing